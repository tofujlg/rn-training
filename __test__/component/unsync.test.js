import React from 'react';
import renderer, { create } from 'react-test-renderer';

import Unsync from '../../component/unsync';

const tree = create(<Unsync/>)

test('snapshot', () => {
  expect(tree).toMatchSnapshot();
});