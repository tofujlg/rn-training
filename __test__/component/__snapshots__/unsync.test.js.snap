// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<View
  style={
    Object {
      "backgroundColor": "orange",
      "borderRadius": 6,
      "elevation": 9,
      "marginHorizontal": 10,
      "marginVertical": 5,
      "padding": 18,
      "shadowColor": "#000",
      "shadowOffset": Object {
        "height": 4,
        "width": 0,
      },
      "shadowOpacity": 0.32,
      "shadowRadius": 5.46,
    }
  }
>
  <Text
    style={
      Object {
        "color": "white",
        "fontWeight": "bold",
        "textAlign": "center",
      }
    }
  >
    機械の接続解除
  </Text>
</View>
`;
